src/Runner/Runner.php

```diff
@@ -109,20 +115,92 @@ public function __construct(
         $this->cacheManager = $cacheManager;
         $this->directory = $directory ?: new Directory('');
         $this->stopOnViolation = $stopOnViolation;
+        $this->parallel = $parallel;
     }
 
     public function fix(): array
     {
-        $changed = [];
+        $finderIterator = $this->finder instanceof \IteratorAggregate
+            ? $this->finder->getIterator()
+            : $this->finder;
 
-        $finder = $this->finder;
-        $finderIterator = $finder instanceof \IteratorAggregate ? $finder->getIterator() : $finder;
         $fileFilteredFileIterator = new FileFilterIterator(
             $finderIterator,
             $this->eventDispatcher,
             $this->cacheManager
         );
 
+        if ($this->parallel > 1) {
+            return $this->fixParallel($fileFilteredFileIterator);
+        }
+
+        return $this->fixProcedural($fileFilteredFileIterator);
+    }
+
+    private function fixParallel(FileFilterIterator $fileFilteredFileIterator): array
+    {
+        if (false === \function_exists('pcntl_fork')) {
+            return $this->fixProcedural($fileFilteredFileIterator);
```

Comment:
throw error maybe? like, you checked can you call `fixParallel` already before, so why to fallback now?